src/Simplex/Messaging/Transport.hs

```diff
@@ -134,11 +141,14 @@ data ATransport = forall c. Transport c => ATransport (TProxy c)
 
 -- * Transport over TLS 1.2
 
+-- | Party name to print on exception handling - for debugging.
+newtype PartyAlias = PartyAlias {partyAlias :: String} deriving (Show)
+
 -- | Run transport server (plain TCP or WebSockets) on passed TCP port and signal when server started and stopped via passed TMVar.
 --
 -- All accepted connections are passed to the passed function.
-runTransportServer :: forall c m. (Transport c, MonadUnliftIO m) => TMVar Bool -> ServiceName -> T.ServerParams -> (c -> m ()) -> m ()
-runTransportServer started port serverParams server = do
+runTransportServer :: forall c m. (Transport c, MonadUnliftIO m) => PartyAlias -> TMVar Bool -> ServiceName -> T.ServerParams -> (c -> m ()) -> m ()
+runTransportServer PartyAlias {partyAlias} started port serverParams server = do
```

Comment:
and now that I see where it is used it seems completely unnecessary too - please remove.
Let's not spend time on such things, it's yak shaving.